Makefile

```diff
@@ -0,0 +1,6 @@
+TAGNAME=bits-and-bytes-bot:latest
+
+.PHONY: docker-build
```

Comment:
Instead of using a makefile would you be opposed to using taskfile? 
https://taskfile.dev/

much easier to use IMO

Reply:
I have not used that but I can convert it.

Reply:
It's really easy to use - should only take a few mins to understand